migrations/versions/6bd24e8f99f0_.py

```diff
@@ -0,0 +1,50 @@
+"""empty message
+
+Revision ID: 6bd24e8f99f0
+Revises: 76c949b1235f
+Create Date: 2017-06-21 20:05:08.127436
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '6bd24e8f99f0'
+down_revision = '76c949b1235f'
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('event_types',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(), nullable=False),
+    sa.Column('slug', sa.String(), nullable=False),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('id'),
+    sa.UniqueConstraint('slug')
+    )
+    op.alter_column('events', 'type', new_column_name='event_type_id')
+    op.alter_column('events_version', 'type', new_column_name='event_type_id')
+    op.execute('INSERT INTO event_types(name, slug) SELECT DISTINCT event_type_id, replace(event_type_id, \' \', \'-\') \
```

Comment:
@niranjan94  If I replace `&` and `,`, won't (for example) 'Camp & Retreat' look like 'Camp---Retreat'? Is it fine?